src/swarm/neo/AddrPort.d

```diff
@@ -25,11 +25,15 @@ public struct AddrPort
     import core.sys.posix.arpa.inet: ntohl, ntohs, htonl, htons, inet_ntop;
     import core.stdc.string: strlen;
 
-    import ocean.util.container.map.model.StandardHash;
+    import swarm.Const : NodeItem;
 
+    import ocean.util.container.map.model.StandardHash;
     import ocean.core.Test;
     import swarm.util.Verify;
 
+    /// Minimum length required for an address format buffer.
+    public static const AddrBufLength = INET6_ADDRSTRLEN;
```

Comment:
`static` is redundant (and wouldn't compile in a `class`)

Reply:
:+1: I'm forgetting what works in D1 and D2. Is it just `const`?

Reply:
Yup, just `const` works.

Reply:
`const` + initializer

Reply:
Ping @gavin-norman-sociomantic :-)